src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -211,7 +211,17 @@ static NodeFunctionInvoker()
             {
                 // get the output value from the script
                 object value = null;
-                if (bindings.TryGetValue(binding.Metadata.Name, out value) && value != null)
+                bool haveValue = bindings.TryGetValue(binding.Metadata.Name, out value);
```

Comment:
This is a bug fix. People generally use the names req/res for http bindings, however this isn't required. I discovered that things would break if you instead used different names like request/response. This fixes that.